recipes/htpolynet/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "htpolynet" %}
+{% set version = "1.0.7.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/htpolynet-{{ version }}.tar.gz
+  sha256: 8c20c631f9bb8634e7bd83338d37502994fdce6692652d01410b5f330b25e2f5
+
+build:
+  entry_points:
+    - htpolynet = HTPolyNet.driver:cli
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - hatchling
+    - pip
+  run:
+    - python >=3.9
+    - numpy >=1.24
+    - pyyaml >=6
+    - requests >=2.28
+    - pandas >=2
+    - scipy >=1.10
+    - parmed >=4
+    - networkx >=2.8
+    - gputil >=1.4
+
+test:
+  imports:
+    - htpolynet
```

Looks like the import name doesn't follow Python naming standards and it capitalized.

```suggestion
    - HTPolyNet
```